src/main/java/org/kpmp/geneExpressionSummary/GeneExpressionSummaryService.java

```diff
@@ -88,4 +95,27 @@ public GeneExpressionSummaryService(SCRNAGeneExpressionSummaryRepository scrnaGe
 		return dataTypes;
 	}
 
+	public List<DatasetSummary> getGeneDatasetInformation(String geneSymbol) {
+		List<DatasetSummary> geneSummary = new ArrayList<>();
+
+		geneSummary.add(new DatasetSummary(
+			"TRANSCRIPTOMICS",
+		 	"Single-cell RNA-seq (scRNA-seq)",

```

Comment:
I wouldn't have called this out, but since I am asking for some other changes, can we move this string into the DataType Enum?